examples/python/src/authors/query.py

```diff
@@ -0,0 +1,92 @@
+# Code generated by sqlc. DO NOT EDIT.
+from typing import AsyncIterator, Awaitable, Iterator, Optional, overload
+
+import sqlc_runtime as sqlc
+
+from authors import models
+
+
+CREATE_AUTHOR = """-- name: create_author :one
+INSERT INTO authors (
+          name, bio
+) VALUES (
+  $1, $2
+)
+RETURNING id, name, bio
+"""
+
+
+DELETE_AUTHOR = """-- name: delete_author :exec
+DELETE FROM authors
+WHERE id = $1
+"""
+
+
+GET_AUTHOR = """-- name: get_author :one
+SELECT id, name, bio FROM authors
+WHERE id = $1 LIMIT 1
+"""
+
+
+LIST_AUTHORS = """-- name: list_authors :many
+SELECT id, name, bio FROM authors
+ORDER BY name
+"""
```

Comment:
I like to generate the query strings closed to the functions themselves so it's easier to read. If I'm looking at `list_authors`, I can then see the query it's referencing without having to jump around.

Reply:
I agree it's nice to keep them close together, but how would this work if the query functions are in a class? I could define these constants as class properties above each method, but I can't say I've seen other Python code that does that, class properties usually go at the top of the class definition.